application/backend/src/main/resources/apps/typerefinery/components/widgets/chart/dialog/.content.json

```diff
@@ -0,0 +1,70 @@
+{
+  "sling:resourceType": "wcm/dialogs/dialog",
+  "tabs": {
+    "sling:resourceType": "wcm/dialogs/components/tabs",
+    "general": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "General",
+      "chartVariant": {
+        "sling:resourceType": "wcm/dialogs/components/select",
+        "name": "chartVariant",
+        "label": "Choose Graph Variants",
+        "lineChart": {
+          "sling:resourceType": "wcm/dialogs/components/select/selectitem",
+          "label": "Line Chart",
+          "value": "lineChart",
+          "selected": "true"
+        },
+        "barChart": {
+          "sling:resourceType": "wcm/dialogs/components/select/selectitem",
+          "label": "Bar Chart",
+          "value": "barChart"
+        },
+        "pieChart": {
+          "sling:resourceType": "wcm/dialogs/components/select/selectitem",
+          "label": "Pie Chart",
+          "value": "pieChart"
+        },
+        "lineChartWithoutLabel": {
+          "sling:resourceType": "wcm/dialogs/components/select/selectitem",
+          "label": "Line Chart(Without Label)",
+          "value": "lineChartWithoutLabel"
+        },
+        "barChartHorizontal": {
+          "sling:resourceType": "wcm/dialogs/components/select/selectitem",
+          "label": "Bar Chart(Horizontal)",
+          "value": "barChartHorizontal"
+        },
+        "pieChartNonPolar": {
+          "sling:resourceType": "wcm/dialogs/components/select/selectitem",
+          "label": "Pie Chart(Non Polar)",
+          "value": "pieChartNonPolar"
+        }
+      }
+    },
+    "dataTab": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "Data",
+      "dataSource": {
+        "sling:resourceType": "wcm/dialogs/components/textfield",
+        "name": "dataSource",
+        "description": "Please provide url to data source for this ticker.",
+        "label": "Data Source"
+      }
+    },
+    "websocketTab": {
```

Comment:
can you please update this to include /apps/typerefinery/components/dialog/websocket.

Reply:
Ok, sure.